bin/sam-translate.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python2
+
+"""Convert SAM templates to CloudFormation templates.
+Known limitations: cannot transform CodeUri pointing at local directory.
+
+Usage:
+  sam-translate.py --input-file=sam-template.yaml [--output-file=<o>]
+
+Options:
+  --input-file=<i>     Location of SAM template to transform.
+  --output-file=<o>    Location to store resulting CloudFormation template [default: cfn-template.json].
+
+"""
+import os
```

Comment:
Not sure if we're trying to follow [PEP8 imports style guide](https://www.python.org/dev/peps/pep-0008/#imports), but if we are, the imports should be separated into 3 groups.

> Imports should be grouped in the following order:
>
> 1. Standard library imports.
> 1. Related third party imports.
> 1. Local application/library specific imports.
>
> You should put a blank line between each group of imports.